### PR TITLE
Adding network ad support

### PR DESCRIPTION
**Pull request by yaustar**
Extra code needs to be added to the top of the index.html file

### CODE REVIEW

File: config.template.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
             "external_url_prefix": ""
         },
         "mraid_support": false,
+        "snapchat_cta": false
```
Comment (yaustar) [inline config.template.json:37]:
```suggestion
        "snapchat_cta": false,
```

File: library-files/snapchat-cta.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+window.snapchatCta = function() {
+    console.log('Snapchat CTA clicked');
+    if (window.mraid) {
+        mraid.open('{{ .ClickTrackingUrl }}');
+    }
+}
```
Comment (yaustar) [inline library-files/snapchat-cta.js:6]:
```suggestion
}

```

**Comment by yaustar** [comment]
@willeastcott Can you review when you get chance please?

File: README.md [hunk 3]
```diff
@@ -170,10 +170,18 @@ In some cases with ad networks, the external files will need to be hosted elsewh
...
 #### MRAID interstitial support
 
-Adds basic support for MRAID API within the PlayCanvas engine and boilerplate code. 
+Adds basic support for MRAID API within the PlayCanvas engine and boilerplate code.
 
 The option can be found in `config.json` under `one_page`. Set `mraid_support` to true.
 
+#### Snapchat ad support
+
+The Snapchat ad network requires the CTA function to be in the `index.html` where the network can replace the with a unique tracking version when it is served to the user. The URL will be set in the Snapchat Ad campaign tool.
```
Comment (willeastcott) [inline README.md:179]:
```suggestion
The Snapchat ad network requires the CTA function to be in the `index.html` where the network can replace it with a unique tracking version when it is served to the user. The URL will be set in the Snapchat Ad campaign tool.
```

**Review by willeastcott — APPROVED** [review]

**Comment by willeastcott** [comment]
Approved with a couple of comments.

File: library-files/snapchat-cta.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+window.snapchatCta = function() {
+    console.log('Snapchat CTA clicked');
```
[the remark this answers is not in the document]
Comment (yaustar) [reply library-files/snapchat-cta.js:2]: I did as a way to be able to test the CTA is working in the Ad. As Snapchat currently has no way to test an ad, it's not possible to know if the logic to the CTA is working easily.

**Pull request merged**